src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx

```diff
@@ -65,18 +65,18 @@ Steps for reviewing and updating your users:
-* If you want to see a view over a long time, go to the time picker on the right side of the page, where it says **Since 30 days ago** and select a different time range.
-* To see more detail about ingest, scroll down on the data ingestion UI page to see an ingest breakdown.
-* We currently don't offer the ability to drill down into every type of data ingest. To do more in-depth analysis, use NRQL queries. Here's a query example of breaking down [`Metric` data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#dimensional-metrics) by top 10 metric names. Note that some metrics are not data ingest metrics and are not billable. To run different analyses, you might also try faceting by `appName` or `Host`.
+* Use our [data ingest cost estimator](/docs/data-apis/manage-data/calculate-data-ingest). 
+* From the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary/#account-dropdown), click **Manage your data**, then click **Data ingestion**. This page gives you an overview of your current data ingestion over the last 30 days. More details on using this page: 
+  * If you want to see a view over a long time, go to the time picker on the right side of the page, where it says **Since 30 days ago** and select a different time range.
+  * To see more detail about ingest, scroll down on the data ingestion UI page to see an ingest breakdown.
+* We currently don't offer the ability to drill down into every type of data ingest. To do more in-depth analysis than the UI provides, use NRQL queries. Here's a query example of breaking down [`Metric` data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#dimensional-metrics) by top 10 metric names. Note that some metrics are not data ingest metrics and are not billable. To run different analyses, you might also try faceting by `appName` or `Host`.
    ```
 FROM Metric SELECT bytecountestimate()/10e8 as 'GB Estimate' 
 SINCE 7 DAYS AGO 
 FACET metricName LIMIT 10 TIMESERIES 1 day
    ```
   For more usage-related NRQL queries, see [Usage queries and alerts](/docs/accounts/accounts-billing/new-relic-one-pricing-users/usage-queries-alerts/).  
-* We also have [a catalog of open source New Relic One apps](https://opensource.newrelic.com/nerdpacks/). One of these apps is [**Quickstarts**](https://opensource.newrelic.com/projects/newrelic/nr1-quickstarts), which lets you deploy pre-built dashboards. One of the dashboards gives you a visualization of your data ingest.
-
+* We also have [a catalog of open source New Relic One apps](https://opensource.newrelic.com/nerdpacks/). One of these apps is a [Data ingestion breakdown app](https://developer.newrelic.com/instant-observability/account-data-ingestion/51e66f0d-153c-493c-a75c-a386c3f170ce). 
```

Comment:
First sentence here seems unneeded. Maybe remove and just reword the second sentence.